src/stripe-payment.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * `stripe-payment`
+ * `Payment broker for stripe`
+ *
+ * @demo demo/index.html
+ * @customElement `stripe-payment`
+ */
+import { LitElement, html, css } from "lit-element/lit-element.js";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+class StripePayment extends LitElement {
+  /**
+   * Convention
+   */
+  static get tag() {
+    return "stripe-payment";
+  }
+  /**
+   * LitElement render styles
+   */
+  static get styles() {
+    return [
+      css`
+        [hidden] {
+          display: none !important;
+        }
```

Comment:
let's add line breaks after rules,

the better to navigate with vim `{` and `}`